core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/spreadsheet/SpreadSheetCollector.java

```diff
@@ -15,34 +16,34 @@
 /**
  * Collect a stream of string rows into a spreadsheet.
  */
-public abstract class SpreadSheetCollector<Workbook, Sheet, Row>
-    implements Collector<String[], SpreadSheetCollector<Workbook, Sheet, Row>.State, byte[]> {
-  public class State {
+public abstract class SpreadSheetCollector<T, Workbook, Sheet, Row>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this generic name to match the regular expression '^[A-Z][0-9]?$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00119)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this generic name to match the regular expression '^[A-Z][0-9]?$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00119)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this generic name to match the regular expression '^[A-Z][0-9]?$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00119)